.gitignore

```diff
@@ -20,4 +20,5 @@ test-output
 
 # Gradle
 /*/.gradle
-/maptruct-iterable-to-non-iterable/target/
\ No newline at end of file
+/maptruct-iterable-to-non-iterable/target/
```

Comment:
use just `target/` instead and remove all the added files from the target directory of `mapstruct-updatemethods-1`.